README.md

```diff
@@ -63,18 +61,6 @@ There is an `invoke` task for running tests, run `pipenv run invoke test`.
 Or run the command `ENV=test python -m pytest` to run the tests on the command
 line.
 
-# Generating private and public keys
-
-To generate the PEM keys used for the signing of data:
-
-    ssh-keygen -t rsa -b 4096 -q -N "" -m PEM -f RS256.key
-    openssl rsa -in RS256.key -pubout -outform PEM -out RS256.key.pub
-
-Base64 encode the public and private keys
```

Comment:
I was referring just to the Base64 bit when I suggested removing it :)

Reply:
Ah! That wen past me :)